GitHub config flow (#21228)
home-assistant/home-assistant.github.io: source/_integrations/github.markdown

---
title: GitHub

> description: How to integrate the GitHub into Home Assistant.

Reviewer: ```suggestion
description: How to monitor GitHub repositories with Home Assistant.
```

At a minimum the word "the" should be removed.  But even then, seems strange to say "integrate github" since it doesn't do commits or pulls or anything.

  - '@ludeeus'
---

The GitHub integration allows you to monitor your favorite [GitHub][github] repositories.

{% include integrations/config_flow.md %}

> When you setup this integration, you will first be guided to allow the integration to use the [GitHub API][github_api] on your behalf. If you do not yet have a [GitHub][github] account you will be promoted to create one during the configuration of the integration.

Reviewer 2: ```suggestion
When you set up this integration, you will first be guided to allow the integration to use the [GitHub API][github_api] on your behalf. If you do not yet have a [GitHub][github] account you will be prompted to create one during the configuration of the integration.
```

Author: Thanks 👍

> When you have authorized the integration, you select repositories you want to monitor, the list of repositories contain repositories you have [starred][github_starred] on GitHub with your account.

Reviewer 2: ```suggestion
When you have authorized the integration, you select repositories you want to monitor, the list contains repositories you have [starred][github_starred] on GitHub with your account.
```

Author: Thanks 👍
